databricks/koalas/tests/test_indexes.py

```diff
@@ -961,6 +958,21 @@ def test_monotonic(self):
                 self.assert_eq(kmidx.is_monotonic_increasing, expected_increasing_result)
                 self.assert_eq(kmidx.is_monotonic_decreasing, pmidx.is_monotonic_decreasing)
 
+        # The datas below cannot be an arguments for `MultiIndex.from_tuples` in pandas >= 1.1.0.
+        # Refer https://github.com/databricks/koalas/pull/1688#issuecomment-667156560 for detail.
+        datas = []
+        datas.append([(-5, None), (-4, None), (-3, None), (-2, None), (-1, None)])
+        datas.append([(None, "e"), (None, "c"), (None, "b"), (None, "d"), (None, "a")])
+        datas.append([(None, None), (None, None), (None, None), (None, None), (None, None)])
```

Comment:
How about just surrounding these with if-block above instead of moving to the end?